client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -163,10 +180,64 @@ export default function CampaignItemDetails( props: Props ) {
 		conversion_last_currency_found,
 	} = campaign_stats || {};
 
+	const getChartStartDate = ( dateRange: ChartSourceDateRanges ) => {
```

Comment:
I'd like to refactor these changes, so that they are in their own file, but in the interest of time, I'm going to follow up with that.